### ISSUE TITLE
Create a CHS-jsPsych exit survey plugin

### ISSUE THREAD

**Issue by becky-gilbert** [lookit/lookit-api#1296]
# TL;DR

We need to create a CHS-jsPsych exit survey.

# Narrative

CHS studies are typically required to include an exit survey. As a researcher, I need an easy way to include the standard CHS exit survey, ideally without much custom programming or large chunks of copy/pasted code.

As CHS developers, we need the exit survey to include certain questions and options. These need to be in a particular format so that the CHS website can retrieve important aspects of their survey response (e.g. data sharing choices, request to withdraw) from the participant's study data.

As a CHS family, I'd like to see the same format and content at the end of each study, so that I can quickly and easily understand the questions and options. 

# Acceptance Criteria

- [x] Tests
- [ ] Documentation
- [x] Scoping: decide how similar this plugin needs to be to the [EFP exit-survey frame](https://www.jspsych.org/7.3/plugins/survey/) in terms of content and functionality, including whether this plugin needs to include the debriefing content to be shown on the next page
- [x] Plugin must present the required questions: child birthdate confirmation, Databrary sharing, video sharing, withdrawl, feedback
- [x] Plugin must offer EFP parameters for including/removing various bits of content
- [ ] Exit survey plugin data must be compatible with the Lookit-API's expectations

# Implementation Notes

jsPsych has a [`survey` plugin](https://www.jspsych.org/7.3/plugins/survey/), which is a wrapper for [SurveyJS](https://surveyjs.io/).

The Lookit-API expects to receive the exit-survey data in a particular format so that it can easily parse certain pieces of information, such as data sharing choices. We need to ensure that the data from this plugin is compatible with the Lookit-API, for instance by restructuring the data when it is stored in the jsPsych data object, and/or modifying the Lookit-API's handling of the exit survey data.

**Comment by becky-gilbert** [comment]
Completed via https://github.com/lookit/lookit-jspsych/pull/24. 

The lookit-jspsych exit-survey plugin meets the AC above, with some caveats:
- **Documentation**: we still need to create a lookit-jspsych docs site, so we will put all of the docs into a separate issue/PR (although we do have JSDom-formatted comments in the code, which will serve as a basis for certain parts of the documentation).
- **Data compatible with lookit-api**: the exit-survey trial data _should_ be compatible, since it contains the same property names/values that come out of the EFP exit survey. There may still be some data compatibility issues, but those will probably be things that we need to figure out once we start modifying the lookit-api to work with jsPsych data, and any issues probably won't be specific to the exit-survey plugin.
